icu4c/source/test/cintltst/ulocbuildertst.c

```diff
@@ -1674,6 +1679,62 @@ static void TestSetLocale() {
     ulocbld_close(bld2);
 }
 
+static void TestBuildULocale() {
+    ULocaleBuilder* bld1 = ulocbld_open();
+    UErrorCode status = U_ZERO_ERROR;
+
+    ulocbld_setLanguage(bld1, "fr", -1);
+    ULocale* fr = ulocbld_buildULocale(bld1, &status);
+    if (assertSuccess(WHERE "ulocbld_buildULocale()", &status)) {
+        assertEquals(WHERE "ulocale_getLanguage()", "fr", ulocale_getLanguage(fr));
+    }
+
+    ulocbld_setLanguage(bld1, "ar", -1);
+    ulocbld_setScript(bld1, "Arab", -1);
+    ulocbld_setRegion(bld1, "EG", -1);
+    ulocbld_setVariant(bld1, "3456-abcde", -1);
+    ulocbld_setUnicodeLocaleKeyword(bld1, "nu", -1, "thai", -1);
+    ulocbld_setUnicodeLocaleKeyword(bld1, "co", -1, "stroke", -1);
+    ulocbld_setUnicodeLocaleKeyword(bld1, "ca", -1, "chinese", -1);
+    ULocale* l = ulocbld_buildULocale(bld1, &status);
+
+    if (assertSuccess(WHERE "ulocbld_buildULocale()", &status)) {
+        assertEquals(WHERE "ulocale_getLanguage()", "ar", ulocale_getLanguage(l));
+        assertEquals(WHERE "ulocale_getScript()", "Arab", ulocale_getScript(l));
+        assertEquals(WHERE "ulocale_getRegion()", "EG", ulocale_getRegion(l));
+        assertEquals(WHERE "ulocale_getVariant()", "3456_ABCDE", ulocale_getVariant(l));
+        assertEquals(WHERE "ulocale_getLanguage()", "ar", ulocale_getLanguage(l));
```

Comment:
Why the test for `locale_getLanguage()` twice?

Reply:
mistake. forgot to remove . Removed now.